lms/static/scripts/frontend_apps/components/dashboard/DashboardActivityFilters.tsx

```diff
@@ -53,6 +54,19 @@ export type ActivityFilterItem<T extends Course | Assignment> = {
 export type DashboardActivityFiltersProps = {
   courses: ActivityFilterSelection | ActivityFilterItem<Course>;
   assignments: ActivityFilterSelection | ActivityFilterItem<Assignment>;
+
+  /**
+   * The segments filter is relevant only when listing students, so not
+   * providing this will prevent that filter to be rendered or taken into
+   * consideration in any way.
+   * As opposed to other filters, the list of entries is not loaded separately,
+   * but provided here.
```

Comment:
```suggestion
   * Unlike other filters, the entries are not loaded by this component but provided
   * as part of the props.
```

I think it was potentially unclear what "separately" meant.